HTML/mockup-axit/js/elements.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef */
+export const elements = {
+  nameTrialForm: document.querySelector('.form-trial__name'),
```

Comment:
sometimes browser api can make your life easier
there is an api to work with forms and form elements by their names 
https://www.w3schools.com/jsref/coll_doc_forms.asp
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name